il.cpp

```diff
@@ -1060,15 +1065,22 @@ bool GetLowLevelILForInstruction(Architecture* arch, uint64_t addr, LowLevelILFu
 						il.Flag(IL_FLAG_C),
 						instr.operation == ARM64_SBCS ? IL_FLAGWRITE_ALL : 0)));
 		break;
-	case ARM64_SBFX:
-	case ARM64_SBFM:
+	case ARM64_SBFIZ:
 		il.AddInstruction(il.SetRegister(REGSZ(operand1), REG(operand1),
```

Comment:
This is (hopefully) not a mistake. It seems that the original incorrect implementation of ARM64_SBFX was actually a correct implementation of ARM64_SBFIZ.